roles/aws/delete-instance/tasks/getcreds.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "Retrieve credentials from env vars"
+  ignore_errors: true
+  no_log: True
+  set_fact:
+    aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+    aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+
+- debug:
```

Comment:
Couldn't these debug prints just be using the `fail` module? Seems more appropriate. 
https://docs.ansible.com/ansible/latest/modules/fail_module.html

Reply:
I borrowed this from another role.  Don't remember which one but have changed for the relevant roles.